java/build-android.gradle

```diff
@@ -137,8 +144,9 @@ publishing {
 			artifact sourcesJar
 
 			pom {
-				name = enableTrainingApis ? 'onnxruntime-training' : 'onnx-runtime'
-				description = enableTrainingApis ? trainingDescription : defaultDescription
+				name = addQnnDependency ? 'onnx-runtime-qnn-test' : (enableTrainingApis ? 'onnxruntime-training' : 'onnx-runtime')
```

Comment:
Assuming this is temporary. When it gets finalized 'onnxruntime-qnn' would be more consistent with how we generally name things.

Reply:
Everything we have published cannot be deleted or replaced.  So you don't need to have this concern. However, it could be a problem if you publish a test package to a formal feed(maven.org) with a future release version string( like 1.20.0). Then it will prevent us publish the release.

Reply:
@sheetalarkadam and I are discussing whether we should publish a test package to maven.org before our release process starts. If not, we don't need to pick another name here.

Reply:
You could publish to the OSSRH SNAPSHOT repository as a test. We publish TF-Java dev binaries there and it uses the same procedure as Maven Central.